docs/examples/integration-test.tsx

```
@@ -0,0 +1,239 @@
+import React from 'react';
```

ESLint

[error] Definition for rule '@typescript-eslint/ban-types' was not found.

(@typescript-eslint/ban-types)